backend/packages/db-backup-restore/index.js

```diff
@@ -0,0 +1,28 @@
+// @flow
```

Comment:
These binaries are copied from the CentOS release of ArangoDB and should be updated along with the database engine.